src/pages/SetPasswordPage.js

```diff
@@ -119,17 +118,30 @@ class SetPasswordPage extends Component {
                                 {this.props.account.error}
                             </Text>
                         )}
+                        {this.props.isSmallScreenWidth && (
+                            <View style={[styles.mt5, styles.mb5]}>
+                                <Image
+                                    resizeMode="contain"
+                                    style={[styles.signinWelcomeScreenshot]}
+                                    source={welcomeScreenshot}
+                                />
+                            </View>
+                        )}
```

Comment:
Not sure if this is the effect you are going for, but I noticed that if we are viewing this on an iPad this "welcome screenshot" is no longer present.

We are using the "narrow" view on all iOS devices so this ends up looking like this...

**iPad**
![Screen Shot 2021-03-26 at 8 36 24 AM](https://user-images.githubusercontent.com/32969087/112678250-c9b99080-8e0e-11eb-8481-8d04962508d9.png)

**iPhone**
![Screen Shot 2021-03-26 at 8 38 38 AM](https://user-images.githubusercontent.com/32969087/112678244-c6bea000-8e0e-11eb-9685-c7243f6fb3e3.png)

Reply:
Ahhh that is strange and definitely not what I was going for. So `isSmallScreenWidth` is dependent on the user agent, and not the actual screen width in pixels.

Reply:
@marcaaron I noticed that this issue appears when you try to run it natively on iOS; I was wondering how you were able to access the set password page on iOS in this branch? Normally for web I access it via the URL.

Reply:
yes, `isSmallScreenWidth` is just based on the screen width on any platform

>this issue appears when you try to run it natively on iOS

I'm using deep links to open this screen directly like this

```
npx uri-scheme open expensify-cash://setpassword/123456 --ios
```

Reply:
@marcaaron Hmm tried it on my end with this branch and I wasn't able to reproduce the bug:

![Simulator Screen Shot - iPad Pro (9 7-inch) - 2021-03-29 at 10 28 07](https://user-images.githubusercontent.com/31285285/112938411-22b15d80-915c-11eb-8c39-7d0860d5d263.png)

Is the main issue you're concerned about the image not showing up for you? Or did you want to change it so that we use the "wide" view for iPad?

Reply:
The screenshot you are showing is the web version of the app shown in Safari app. The issue I am describing occurs in the native version of the app. Maybe try merging the `main` branch back into this branch and try again.

Reply:
@marcaaron 

Ah! I uploaded the wrong screenshot (I've got like 50 screenshots on my desktop, my bad!)

<img width="716" alt="Screen Shot 2021-03-30 at 1 30 27 PM" src="https://user-images.githubusercontent.com/31285285/113078154-1e8d4a80-9205-11eb-964a-904650046013.png">

Reply:
@marcaaron 

So the issue is that we're defaulting to the narrow view on all iOS devices, but this iOS device in particular has a width greater than the cut off for mobile devices. 

Since we put everything into a single component, I have to display the welcome screenshot conditionally based on the screen width.

I tried modifying `index.native.js` to use the screenWidth conditionally to display the narrow/wide views, but the wide view doesn't display correctly on iOS.

<img width="776" alt="Screen Shot 2021-03-31 at 12 22 11 PM" src="https://user-images.githubusercontent.com/31285285/113090157-b9911f00-921b-11eb-8b80-3bd9d33addbb.png">


A solution is to refactor everything to use the same pattern as the log in page, or to figure out why the styles aren't showing up correctly for the wide view on iOS (which could potentially be a deep rabbit hole).

Reply:
I've refactored everything back into multiple components seeing that the sign in page follows this same pattern, and it makes more sense to have two pages that look nearly identical to have the same structure. This is necessary for everything to display properly seeing that we default to the narrow view for ios.

<img width="769" alt="Screen Shot 2021-03-31 at 1 11 01 PM" src="https://user-images.githubusercontent.com/31285285/113093459-8c943a80-9222-11eb-950e-c33ed8358385.png">